airflow/contrib/hooks/slack_webhook_hook.py

```diff
@@ -95,6 +95,12 @@ def _get_token(self, token, http_conn_id):
             raise AirflowException('Cannot get token: No valid Slack '
                                    'webhook token nor conn_id supplied')
 
+    def _get_base_url(self):
```

Comment:
This function is never called from anywhere outside of the test you have added - We don't need this do we?

Reply:
This function is just to check if the `base_url` is correct. I can remove this one.